securedrop_client/api_jobs/base.py

```diff
@@ -60,20 +59,12 @@ def _do_call_api(self, api_client: API, session: Session) -> None:
             try:
                 self.remaining_attempts -= 1
                 result = self.call_api(api_client, session)
-            except AuthError as e:
-                raise ApiInaccessibleError() from e
-            except ApiInaccessibleError as e:
-                # Do not let ApiInaccessibleError emit the regular failure signal, raise now
+            except (ApiInaccessibleError, AuthError) as e:
                 raise ApiInaccessibleError() from e
-            except RequestTimeoutError:
-                logger.debug('Job {} timed out'.format(self))
+            except Exception as e:
```

Comment:
one thought: doesn't this have us retrying all jobs `DEFAULT_NUM_ATTEMPTS` times, even if they didn't fail due to a timeout? We really only want to be retrying jobs that failed due to network issues (otherwise we will be doing work we _know_ will fail), e.g. in the case of an action that fails due to the object being deleted (404)